README.md

```diff
@@ -1,3 +1,6 @@
+## Maintenance Mode
+**The code in this repo is in maintenance mode, with limited plans for future changes. If you have an issue with the current codebase, we encourage you to try out [`vue-cli` 3.0](https://github.com/vuejs/vue-cli/blob/dev/README.md), using its [new `pwa` plugin](https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-pwa), which is being actively developed.**
```

Comment:
I might wordsmith this a little. e.g s/If you have an issue with the current codebase../Our focus moving forward is adding PWA support to Vue CLI 3.0, using its new pwa plugin. We encourage this work to be used for new projects as it is being actively developed. If you have issues with the current codebase, give the new work a spin and let us know what you think.

(feel free to riff) I think the main change I'd suggest is moving away from the newer work you and Vue have done as just being worth checking out if there are issues with the current codebase.

Reply:
Sounds good to me.